discovery/pkg/apigee/pollproxiesjob_test.go

```diff
@@ -119,11 +120,13 @@ func Test_pollProxiesJob(t *testing.T) {
 				crds := []string{}
 				if tc.hasAPIKey {
 					crds = append(crds, provisioning.APIKeyCRD)
+					allowedOAuthMethods = append(allowedOAuthMethods, provisioning.APIKey)
```

Comment:
should not be necessary

Reply:
allowedCRDs?

Reply:
I guess the better fix for this is to drop the var and send in an empty array to the GetCred...method.

Reply:
done...